indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -27,20 +28,34 @@ export default function FormItem({
 }) {
   const meta = fieldRegistry[inputType] || {};
   const InputComponent = meta.inputComponent;
+  const {customFormItem} = meta;
```

Comment:
I'd use `meta.customFormItem` directly; we don't destructure the other data in `meta` either and mixing both in the same block is a bit ugly.